pkg/config/engine/docker/docker.go

```diff
@@ -103,6 +103,11 @@ func (c Config) DefaultRuntime() string {
 	return r
 }
 
+// EnableCDI sets features.cdi to true in the docker config.
+func (c *Config) EnableCDI() {
+	c.Set("features", map[string]bool{"cdi": true})
```

Comment:
Out of scope of this PR: I've just thought about the fact that we would overrwrite existing features here.

Reply:
I see you fixed this in a later commit.